src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2924,9 +2924,9 @@ public override IEnumerable<ISymbol> GetSymbolsWithName(Func<string, bool> predi
 
         #endregion
 
-        internal override AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerOptions options, Func<Exception, DiagnosticAnalyzer, bool> continueOnAnalyzerException, CancellationToken cancellationToken)
+        internal override AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerOptions options, AnalyzerManager analyzerManager, Func<Exception, DiagnosticAnalyzer, bool> continueOnAnalyzerException, CancellationToken cancellationToken)
```

Comment:
continueOnAnalyzerException func, can we not pass around random func to change behavior? either we should use hierarchy so certain type of analyzer driver has same exception handling behavior or use composable/strategy pattern to change certain behavior. with random func, it is so easy to lose control what is going on.